modules/gridmap/doc_classes/GridMapEditorPlugin.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GridMapEditorPlugin" inherits="EditorPlugin" keywords="tilemap" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		Editor for [GridMap] nodes.
+	</brief_description>
+	<description>
+		GridMapEditorPlugin provides access to the [GridMap] editor functionality.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_current_grid_map">
+			<return type="GridMap" />
+			<description>
+				Returns the [GridMap] currently edited by the grid map editor.
+			</description>
+		</method>
+		<method name="set_selection">
+			<return type="void" />
+			<param index="0" name="active" type="bool" />
+			<param index="1" name="begin" type="Vector3i" />
+			<param index="2" name="end" type="Vector3i" />
+			<description>
+				Set a rectangular selection from [param begin] to [param end].
+				Use [code]false[/code] for the [param active] parameter to disable the selection.
```

Comment:
I am to assume it actually works like this
```suggestion
				Adds the cells inside the bounding box created from the [param begin] and [param end] points to the current selection.
				If [param active] is [code]false[/code], the cells inside the box are subtracted from the selection, instead.
```
Or, perhaps easier to digest:
```suggestion
				Adds the cells inside the given bounds to the current selection. The bounds are defined by the [param begin] and [param end] points.
				If [param active] is [code]false[/code], the cells inside the box are subtracted from the selection, instead.
```
Or similar.

I also personally think `active` should be moved to the end as an optional argument and given a more descriptive name.

Reply:
The selection changes to the box defined by the begin and end, it doesn't add anything. "Add" would imply that you can have multiple areas selected, but this isn't currently supported (it is something to be added later, so i also try to keep backwards compatibility in mind with what is exposed).

Perhaps something like "change the selection to a box from begin to end"?

Active doesn't mean add/subtract (the grid map editor currently only supports a box as a selection) but that there is a selection that can be used. I also think `active` isn't a good name (and i'd rather have something like `clear_selection` as a separate method) but it is what the rest of the code uses (actually the entire method just forwards the call to another with the exact same name and parameters, so i decided to leave it as is).

Reply:
Oh, that's pretty... nasty. In that case:
```suggestion
				Selects the cells inside the given bounds from [param begin] to [param end].
				If [param active] is [code]false[/code], clears the current selection.
```

Reply:
You know, there is no reason to keep the exact same arguments here, after i fix the Mono build, i'll just remove `active` from the exposed method and have it always pass `true` (i can't think a reason to ever want to pass both `false` and coordinates), rename `is_selection_active` to `has_selection` and add a `clear_selection` that sets an empty selection and passes `false` for `active`.

The selection system will need to be refactored in the future to support arbitrarily shaped selections anyway.

Reply:
I did consider using "bounds" when writing that but i was worried that it might confused with bounds in 3D space when the coordinates you pass are meant to be cell coordinates inside the grid. Though perhaps passing `Vector3i` (i.e. integers) might avoid that confusion.

Reply:
Changed the API and updated the docs as mentioned (removed active too).